app/code/Magento/Bundle/Model/LinkManagement.php

```diff
@@ -90,7 +97,8 @@ public function __construct(
         CollectionFactory $optionCollection,
         StoreManagerInterface $storeManager,
         DataObjectHelper $dataObjectHelper,
-        MetadataPool $metadataPool
+        MetadataPool $metadataPool,
+        Link  $linkHelper
```

Comment:
I suggest to make it nullable to prevent backward-incompatibility in inherited classes.
```suggestion
        ?Link  $linkHelper = null
```

Reply:
Faced with tests failing with it. Need to investigate this moment.

Reply:
Please check that in case of null, object should be instantiated via Object Manager (like in \Magento\Framework\App\DocRootLocator).